diff -ur phpPgAdmin-5.1.orig/libraries/lib.inc.php phpPgAdmin-5.1/libraries/lib.inc.php
--- phpPgAdmin-5.1.orig/libraries/lib.inc.php	2013-04-14 22:59:58.000000000 +0000
+++ phpPgAdmin-5.1/libraries/lib.inc.php	2013-08-05 07:57:03.087116156 +0000
@@ -10,7 +10,7 @@
 	include_once('./lang/translations.php');
 
 	// Set error reporting level to max
-	error_reporting(E_ALL);
+	error_reporting(E_ALL & ~E_NOTICE);
  
 	// Application name
 	$appName = 'phpPgAdmin';
@@ -27,12 +27,12 @@
 		exit(sprintf('Version of PHP not supported. Please upgrade to version %s or later.', $phpMinVer));
 
 	// Check to see if the configuration file exists, if not, explain
-	if (file_exists('conf/config.inc.php')) {
+	if (file_exists('/etc/webapps/phpPgAdmin/config.inc.php')) {
 		$conf = array();
-		include('./conf/config.inc.php');
+		include('/etc/webapps/phpPgAdmin/config.inc.php');
 	}
 	else {
-		echo 'Configuration error: Copy conf/config.inc.php-dist to conf/config.inc.php and edit appropriately.';
+		echo 'Configuration error: Copy conf/config.inc.php-dist to /etc/webapps/phpPgAdmin/config.inc.php and edit appropriately.';
 		exit;
 	}
 
