--- phpPgAdmin-4.0.1/libraries/lib.inc.php~	2005-12-02 17:19:47.000000000 +0200
+++ phpPgAdmin-4.0.1/libraries/lib.inc.php	2005-12-02 17:24:01.259851274 +0200
@@ -26,12 +26,12 @@
 		exit(sprintf('Version of PHP not supported. Please upgrade to version %s or later.', $phpMinVer));
 
 	// Check to see if the configuration file exists, if not, explain
-	if (file_exists('conf/config.inc.php')) {
+	if (file_exists('/etc/webapps/phpPgAdmin/config.inc.php')) {
 		$conf = array();
-		include('./conf/config.inc.php');
+		include('/etc/webapps/phpPgAdmin/config.inc.php');
 	}
 	else {
-		echo 'Configuration error: Copy conf/config.inc.php-dist to conf/config.inc.php and edit appropriately.';
+		echo 'Configuration error: Copy conf/config.inc.php-dist to /etc/webapps/phpPgAdmin/config.inc.php and edit appropriately.';
 		exit;
 	}
 
--- phpPgAdmin-4.0.1/classes/plugins/Plugin.php~	2005-06-16 17:40:12.000000000 +0300
+++ phpPgAdmin-4.0.1/classes/plugins/Plugin.php	2005-12-02 17:25:54.525481589 +0200
@@ -21,7 +21,7 @@
 		// Read in configuration
 		if ($this->config !== null) {
 			global $conf;
-			include('./conf/' . $name . '.inc.php');			
+			include('/etc/webapps/phpPgAdmin/' . $name . '.inc.php');			
 		}
 	}
 
